src/System.Reflection.Metadata/src/System/Reflection/Metadata/Handles.cs

```diff
@@ -1879,10 +2503,10 @@ public override int GetHashCode()
     public struct StringHandle : IEquatable<StringHandle>
     {
         // bits:
-        //  0..24: Heap index or Virtual index
-        // 25..30: TokenTypeId: String, WinRTPrefixedString
         //     31: IsVirtual
-        private readonly uint _token;
+        // 29..31: type (non-virtual: String, DotTerminatedString; virtual: WinRTPrefixedString)
```

Comment:
Actually, this is not typo :) The type includes the virtual bit. The comment should be
```// 29..31: type (non-virtual: String, DotTerminatedString; virtual: VirtualString, WinRTPrefixedString)``` though